nifi-docker/dockerhub/sh/secure.sh

```diff
@@ -45,13 +45,23 @@ prop_replace 'nifi.security.truststore'         "${TRUSTSTORE_PATH}"
 prop_replace 'nifi.security.truststoreType'     "${TRUSTSTORE_TYPE}"
 prop_replace 'nifi.security.truststorePasswd'   "${TRUSTSTORE_PASSWORD}"
 
+prop_replace 'keystore'           "${KEYSTORE_PATH}"        ${nifi_toolkit_props_file}
+prop_replace 'keystoreType'       "${KEYSTORE_TYPE}"        ${nifi_toolkit_props_file}
+prop_replace 'keystorePasswd'     "${KEYSTORE_PASSWORD}"    ${nifi_toolkit_props_file}
```

Comment:
Sometimes the `keyPasswd` is different from the `keystorePasswd` so probably needs to be included here as well.